OMGVA PoS/Data layer/DTOs/Tax/UpdateTaxRequest.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.ComponentModel.DataAnnotations;
+
+namespace OMGVA_PoS.Data_layer.DTOs
+{
+    public class UpdateTaxRequest
+    {
+        
+        [MinLength(1, ErrorMessage = "TaxType must be at least 1 character.")]
+        public string? TaxType { get; set; }
+        
+        [Range(0, short.MaxValue, ErrorMessage = "Percent must be 0 or greater.")]
+        public short? Percent { get; set; }
+
+        public bool? IsArchived { get; set; }
```

Comment:
Maybe this should be updatable only with delete request?

Reply:
Not sure. What if somebody "deletes" tax by accident and we want to unarchive it. I'd say it should be possible to change this field. 

Also, thanks for reminding me - we decided to never delete taxes and just update this field

Reply:
I think in the example you just gave the user should just create a new tax and don't mess with archivation. Still believe this one is unneccasary. Now that I think about it, what happens to the products that have the tax applied that was archived? We should probably check it and disallow it. It is just something to keep in mind.

Reply:
I'd say we can leave it for now, and decide this detail later once we have more endpoints/implementations done, what do you think?

Reply:
Yes I agree for the second part. Currently, it would not even be possible. Just something to keep in mind. However, I still believe that unarchiving should still not be an option.